academy/4-my-own-chain/create-message.md

```diff
@@ -125,6 +125,94 @@ service Msg {
 
 As an interface it does not describe what should happen when called. What Starport does with the help of Protobuf is compile the interface and create a default Go implementation.
 
+## Interact via the CLI
+
+Time to see which new CLI command was created by Starport:
+
+```sh
+$ checkersd tx checkers --help
+...
+Available Commands:
+  create-game Broadcast message createGame
+```
+
+And also:
+
+```sh
+$ checkersd tx checkers create-game --help
+...
+Usage:
+  checkersd tx checkers create-game [red] [black] [flags]
+
+Flags:
+   -a, --account-number uint      The account number of the signing account (offline mode only)
+   -b, --broadcast-mode string    Transaction broadcasting mode (sync|async|block) (default "sync")
+       --dry-run                  ignore the --gas flag and perform a simulation of a transaction, but don\`t broadcast it
+       --fees string              Fees to pay along with transaction; eg: 10uatom
+       --from string              Name or address of private key with which to sign
+       --gas string               gas limit to set per-transaction; set to "auto" to calculate sufficient gas automatically (default 200000)
+...
+```
+
+You kept the two accounts that were created by Starport. Have `alice` start a game with `bob`. Simplify your life and make them aliases:
+
+```sh
+$ export alice=cosmos1wh7scjfhgzeqxfxhqq6jh59sj2y8d7u97qu7qp
+$ export bob=cosmos199krg6nz4qgv53nvrx9gj7nrlg48clwurn82jy
+```
+
+ How much gas is needed? Estimate that with:
+
+```sh
+$ checkersd tx checkers create-game `echo $alice` `echo $bob` --from `echo $alice` --dry-run
+gas estimate: 40452
+```
+
+That is not much. Thus, `auto` is enough:
```

Comment:
Not entirely sure what the point of this is. What if it was a lot? You still want to use `--gas auto`, no?

Reply:
Let me think about it :)